prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml

```diff
@@ -188,7 +188,7 @@ periodics:
     testgrid-tab-name: autobump-prow
   spec:
     containers:
-    - image: gcr.io/k8s-prow/autobump:v20200628-cc1c099dad
+    - image: gcr.io/k8s-prow/autobump:v20200706-8795b499fa
```

Comment:
Lets avoid bumping past the version that the bump PR suggests: https://github.com/GoogleCloudPlatform/oss-test-infra/pull/419
Otherwise the bump PR could revert the autobump version tag to a previous version which just caused problems for Istio since it was bumped back to a version that was missing a needed fix.

Reply:
Sg, updated

Reply:
@cjwagner PTAL